dashboard-visualiser-superset/package-metadata.json

```
@@ -12,6 +12,14 @@
     "SUPERSET_EMAIL": "admin@superset.com",
     "SUPERSET_PASSWORD": "admin",
     "SUPERSET_CONFIG_FILE": "superset-export.zip",
-    "SUPERSET_SSL": "false"
+    "SUPERSET_SSL": "false",
+    "KC_SUPERSET_SSO_ENABLED": "true",
```

This default should also be false.
```suggestion
    "KC_SUPERSET_SSO_ENABLED": "false",
```

---

done :)